spring-cloud-contract-wiremock/src/main/java/org/springframework/cloud/contract/wiremock/WireMockConfiguration.java

```diff
@@ -97,9 +97,6 @@ private void registerStubs() throws IOException {
 						this.resourceLoader);
 				String pattern = stubs;
 				if (!pattern.contains("*")) {
-					if (!pattern.endsWith("/")) {
```

Comment:
I essentially have my wiremock mappings and stubs in a jar file that is in artifactory.  The stubs are in resources/wiremock/__files and the mappings are in resources/wiremock/mappings.  I am then using: ```@Import(WireMockConfiguration.class)```  along with the following in my application.yml
```
wiremock:
  server:
    files: classpath:wiremock
    stubs: 
    httpsPort: -1
```